apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableByteString.cs

```diff
@@ -0,0 +1,301 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Protobuf;
+using System;
+using System.Text;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    /// <summary>
+    /// The bytes string which can be used for a row key, column qualifier, or cell value.
```

Comment:
"The bytes string" doesn't sound quite right. How about "A string of bytes"? Or "A sequence of bytes"?

Reply:
Oops. That was supposed to be "byte string", but I think "sequence of bytes" sounds better anyway.